src/applications/personalization/dashboard-2/lib/moment-tz.js

```diff
@@ -0,0 +1,1278 @@
+/* istanbul ignore file */
```

Comment:
We are using this to get the moment-timezone functionality we need rather than using moment-timezone directly because that package is so large. VAOS is doing the same thing. In fact, this is a copy of the "library" that VAOS is using. We should look at moving this into `platform`.